web/layout_attributes/using_layout_classes/index.dart

```diff
@@ -3,7 +3,7 @@ import 'my_horizontal_element.dart';
 import 'my_vertical_element.dart';
 
 /// Silence analyzer [MyHorizontalElement], [MyVerticalElement]
-main() async {
+ main() async {
```

Comment:
looks like this got accidentally added in, probably just want to run the formatter over the whole repo anyways and add that as a separate commit.